scripts.js

```diff
@@ -2,4 +2,5 @@ var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that firs
 var dinosaur = 'triceratops';
 var dinosaurUpperCased = dinosaur.toUpperCase();
 var textCharsAfter = text.replace('Velociraptor', dinosaurUpperCased);
-console.log(textCharsAfter.substr(0,72));
+var halfText = textCharsAfter.substr(0,72);
```

Comment:
replace number 72 with variable. this variable should be a calculation of text length